morf-core/src/main/java/org/alfasoftware/morf/jdbc/ConnectionResourcesBean.java

```diff
@@ -430,6 +472,8 @@ public final boolean equals(Object obj) {
       if (other.schemaName != null) return false;
     } else if (!schemaName.equals(other.schemaName)) return false;
     if (statementPoolingMaxStatements != other.statementPoolingMaxStatements) return false;
+    if (fetchSizeForBulkSelects != other.fetchSizeForBulkSelects) return false;
+    if (fetchSizeForBulkSelectsAllowingConnectionUseDuringStreaming != other.fetchSizeForBulkSelectsAllowingConnectionUseDuringStreaming) return false;

```

Comment:
These are boxed Integer values, I am not sure `fetchSizeForBulkSelects != other.fetchSizeForBulkSelects` is correct comparison here. See Objects.equals()